test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -402,8 +401,8 @@ public void getStudyParticipant() {
         assertTrue(retrievedHistory2.isEmpty());
         
         // A non-researcher will not get the healthCode
-        participant = participantService.getParticipant(STUDY, Sets.newHashSet(DEVELOPER), ID);
-        assertNull(participant.getHealthCode());
+        participant = participantService.getParticipant(STUDY, ID, false);
+        assertEquals(HEALTH_CODE, participant.getHealthCode());
```

Comment:
Removed. This tests the same thing as above and was originally because roles were being switched in the test, which is no longer the case.